### DISCUSSION TITLE
Stop button does not working

### DISCUSSION THREAD

**Discussion by tcarafini** [tprouvot/Salesforce-Inspector-reloaded#476] [Q&A] [answered]
Hello, 
First of all I would like to congratulate the developers of Inspector, it is a great working tool.
I have a problem, in version 1.24.3, the stop button is no longer working, can you tell what it could be?

Thanks, 
Tiago Carafini

**Comment by tprouvot** [comment, marked as answer]
Hi @tcarafini,
I think this might be related to the recent updates concerning the SOSL and GraphQL new languages support.
I can reproduce the error and will try to solve it ASAP, I already have to update the beta version since it should be this week

**Reply by tcarafini** [reply]
Hello @tprouvot ,
No problem, just wanted to let you know, in case you haven't seen it. 

Thank you so much for the feedback.

**Reply by tprouvot** [reply]
Thank you for highlighting this ! I wasn't aware of this issue

**Comment by tprouvot** [comment]
Bug fixed in releaseCandidate, I'll update beta version
